t/process-master-pid.t

```diff
@@ -0,0 +1,68 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
+use lib 'lib';
+use Test::Nginx::Socket::Lua;
+use Cwd qw(cwd);
+
+#worker_connections(1014);
+#master_process_enabled(1);
+#log_level('warn');
+
+repeat_each(2);
+
+plan tests => repeat_each() * (blocks() * 6);
+
+my $pwd = cwd();
+
+our $HttpConfig = <<_EOC_;
+    lua_shared_dict dogs 1m;
+    lua_package_path "$pwd/lib/?.lua;../lua-resty-lrucache/lib/?.lua;;";
+    init_by_lua_block {
+        local verbose = false
+        if verbose then
+            local dump = require "jit.dump"
+            dump.on("b", "$Test::Nginx::Util::ErrLogFile")
+        else
+            local v = require "jit.v"
+            v.on("$Test::Nginx::Util::ErrLogFile")
+        end
+
+        require "resty.core"
+        -- jit.off()
+    }
+_EOC_
+
+#no_diff();
+#no_long_string();
+check_accum_error_log();
+run_tests();
+
+__DATA__
+
+=== TEST 1: ngx.process.master_pid
+--- http_config eval: $::HttpConfig
+--- config
+    location = /t {
+        content_by_lua_block {
+            local process = require "ngx.process"
+
+            local v
+            local pid = process.master_pid
```

Comment:
This is definitely wrong. You missed a pair of parentheses after `process.master_pid`. Have you actually run this test yourself?

This is another reason why you should make your PR pass the travis ci checks as @membphis suggested ;)

Reply:
Sorry, I'm not familar with travis ci, I will try to correct it ,thanks.

Reply:
And here is not an error ,because `pid` will be called in then next line ：
`local v = pid()`